docs/reference/msbuild-targets.md

```diff
@@ -366,6 +371,23 @@ Restore creates the following files in the build `obj` folder:
 | `{projectName}.projectFileExtension.nuget.g.props` | References to MSBuild props contained in packages |
 | `{projectName}.projectFileExtension.nuget.g.targets` | References to MSBuild targets contained in packages |
 
+### Restore and build in the same MSBuild invocation
```

Comment:
this section seems to come out of nowhere. Is there an error associated when the user does `msbuild -t:restore,build` ?

Or it should be written as, "Multiple operations in the same MSBuild invocation", then talk about some good examples, and then talk about some bad examples.

Reply:
There are no specific error associated with the above, which is why it's such a big problem. 
You don't know it doesn't work until it's too late. 

It's not really multiple operations during the same msbuild invocation. 

The MSBUild side switch is explicitly added for the restore target. 
Only restore can be invoked with `msbuild -restore -t:SomeTargetName`

Reply:
How about "Restoring and building with one MSBuild command"? Sure, technically the issue is with restoring and any other target in the same command, but I imagine that most people who hit this are building and this title is more likely to catch their eye.

Reply:
I like that.

Reply:
let's run with Andy's suggestion.